Library/Homebrew/dev-cmd/pull.rb

```diff
@@ -605,6 +593,37 @@ def verify_bintray_published(formulae_names)
     end
   end
 
+  def check_publication(f, url, max_retries, retry_count)
+    http = create_http(url)
+    http.start do
+      res = get_http_result(http, url)
+      return true if res.is_a?(Net::HTTPSuccess)
+      if res.is_a?(Net::HTTPRedirection)
+        retry_count += 1
```

Comment:
I'd call it `redirect_count`.

Reply:
I used the existing `retry_count` variable, which is used on the main loop, so I kept the same name for consistency.

In fact I could have a separate `retry_count` and `redirect_count` variable, and have 2 different max retries/redirects, but I am not sure it is worth it. Max values of retries is 300, and I do not expect more than 5 redirects (though I am not a web developer, so I may be wrong).

Reply:
Yes, I noticed the resuse... while I agree it's safe to expect just a few redirects, imagine every retry taking 10 redirects. This way you reduce the number of retries from 300 to 30.